### PR TITLE
Math masters project: Setup

### PR DESCRIPTION
**Pull request by AFakhriddinov**
**In this milestone,**

Installed latest version of Node JS
Created a react app
Setup linters for React/Redux

### CODE REVIEW

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
Hi @AFakhriddinov :wave:, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

invalid

## Invalid Code Review Request

You have submitted a pull request with no correctly setup linters to enforce strict compliance with industry standards by catching bad syntax. Kindly follow [this guide](https://github.com/microverseinc/linters-config/tree/master/javascript) on how to setup your linters well, and submit another PR when you are done.

- _You should have something similar to this image (only that all checks must pass):_
image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by AFakhriddinov** [comment]
Dear @AbayomiOlaoye, thanks for your feedback. The link you provided as a guide is for javascript. Am I not supposed to do React/Redux linters? Also I do not know why linters checks are not showing up. Could you please help me understand it?

**Comment by AbayomiOlaoye** [comment]
Oh, I'm sorry. I am seeing this very late.
 Yeah, it should be React linters. What have you tried, @AFakhriddinov ?

**Comment by AbayomiOlaoye** [comment]
> Dear @AbayomiOlaoye, thanks for your feedback. The link you provided as a guide is for javascript. Am I not supposed to do React/Redux linters? Also I do not know why linters checks are not showing up. Could you please help me understand it?

Most times, it could be because some dependencies are not properly or fully installed. Try following the tips and run all necessary commands on your Command-Line or Terminal

File: README.md [hunk 1]
```diff
@@ -1,2 +1,161 @@
...
+cd Math-masters
+cd my-app
+
+### Install
+
+N/A
+
+### Usage
+
+Use this project in the browsers of desktops
```
Comment (topeogunleye) [inline README.md:109]: ## OPTIONAL

Hi @AFakhriddinov! 😃.  I hope you're doing well. I've been reviewing your code and noticed that your README file doesn't provide enough information for someone to set up and use your project. While it's great that you've listed the required technologies, the instructions provided for set up and usage are not clear enough. 

To make it easier for others to use your project, you should consider providing detailed instructions that walk users through the installation process. For example, you could include information on how to install any necessary dependencies, and run the project. Additionally, you might want to consider including screenshots or videos to make it easier for others to follow along.

👍I believe in you and your ability to improve the instructions in your README file. Remember, providing clear and concise instructions can help make your project more accessible and user-friendly. If you have any questions or need further guidance, please don't hesitate to ask. Keep up the great work!

I came across a good example of set up and usage instructions that you could reference when updating your README file. Here is an example:

```
### Setup
If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.

[Download the ZIP from this location](https://github.com/anasbassi/todo-list/archive/refs/heads/main.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:

```bash
git clone https://github.com/anasbassi/todo-list.git
```
```
- Once the files are on your machine, open the _todo-list_ folder in [Visual Studio Code](https://code.visualstudio.com/), and follow the steps in Installation & Set Up

## 🛠 Installation & Set Up

1. Install Node Modules

   ```sh
   npm install
   ```
```
2. Start the development server

   ```sh
   npm start
   ```
```
3. Open [http://localhost:8080](http://localhost:8080) to view it in the browser.

## 🚀 Building and Running for Production

1. Generate a full static production build

   ```sh
   npm run build
```

I hope this helps you improve the instructions in your README file. If you have any questions or need further guidance, please let me know.

**Review by topeogunleye — APPROVED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi @AFakhriddinov,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟

wiserhood-cheers

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:

## [Optional] suggestions:
One thing I noticed while reviewing your code is that the name of your branch, "setup", doesn't follow the conventional naming scheme. In general, branch names should reflect the action or feature being worked on in that branch. Using descriptive and specific branch names can make it easier for you and your collaborators to understand the purpose of each branch.

👉I would suggest that you consider renaming the branch to something like "improve-setup-instructions" or "added-feature-x" to better reflect the work being done in that branch. This can help improve the organization and clarity of your project.

Let me know if you have any questions or need further assistance!

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear
Please, remember to tag @topeogunleye in your question so I can receive the notification

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**